packages/rich-text-editor/src/RichTextEditor/RichTextEditor.scss

```diff
@@ -56,6 +56,14 @@
   border: $border-solid-border-width $border-solid-border-style $color-gray-500;
   background: $color-white;
   border-radius: $border-solid-border-radius;
+
+  div + .editor :global(.ProseMirror) {
+    &:hover,
+    &:focus {
```

Comment:
I think this can be applied regardless of hover or focus, like it is being done above?

Reply:
Yes @dougmacknz